src/app/components/liquid.js

```diff
@@ -1,19 +1,19 @@
+import countLiquid from '../base/countLiquid';
+
 export default class Liquid {
     constructor(container) {
         this.container = container;
         this.liquidRate = this.container.querySelector('.js-liquidRate');
+        this.liquidLowIcon = this.container.querySelector('.js-liquidLow');
+        this.liquidNormalIcon = this.container.querySelector('.js-liquidNormal');
         this.litres = this.container.querySelector('.health__litres');
         this.optimalLitres = this.container.querySelector('.js-liquid-optimal');
-        this.count();
-    }
+        this.litres.dataset.liquid = 1;
+        this.optimalLitres.dataset.liquid = 1.5;
```

Comment:
Why are you immediately reassigning the value that you get from `data-attribute`?